internal/controllers/user_integrations_auth_controller.go

```diff
@@ -1,13 +1,21 @@
 package controllers
 
 import (
+	"context"
+	"encoding/json"
+	"fmt"
```

Comment:
nit: Should separate the std lib packages from other packages with a newline 
https://go.dev/wiki/CodeReviewComments#imports

Reply:
Is there a linter rule for this?

I think VS Code does this by default (whatever they're running on save) but I think Doyin uses GoLand and it's doing something else./